midtransclient/http_client.py

```diff
@@ -39,14 +39,30 @@ def request(self, method, server_key, request_url, parameters=dict(),
         default_headers = {
             'content-type': 'application/json',
             'accept': 'application/json',
-            'user-agent': 'midtransclient-python/1.2.0'
+            'user-agent': 'midtransclient-python/1.5.0'
         }
         headers = default_headers
 
         # only merge if custom headers exist
         if custom_headers:
             headers = {**default_headers, **custom_headers}
 
+        # raise Midtrans error if server_key is not valid format
+        if server_key == "" or server_key is None:
+            raise MidtransAPIError(
+                message='The ServerKey is invalid, as it is an empty string or Null. Please double-check your API key. You can check the ServerKey from the Midtrans Dashboard. See https://docs.midtrans.com/en/midtrans-account/overview?id=retrieving-api-access-keys for the details or contact support at support@midtrans.com if you have any questions.',
+                api_response_dict=None,
```

Comment:
Review for the enhancement part (to address later after bugfix release):
- I think Bintar's team is starting to deprecate email support in favor of website contact form. Should replace the email with URL instead.

Reply:
Agreed, sure will remove the email